### PR TITLE
Hypershift details

### PR DESCRIPTION
**Pull request by rawagner**
ACM counterpart https://github.com/stolostron/console/pull/1823

### CODE REVIEW

**Review by jgyselov — DISMISSED** [review]
Changes in `/common` look good to me from an OCM POV.

File: src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+                          : undefined
+                      }
+                    />
+                    <Td>{np.metadata?.name}</Td>
+                    <Td>
+                      <Popover
+                        aria-label="node pool hosts"
+                        hasAutoWidth
+                        bodyContent={t('{{agents}} of {{count}} host requested is available', {
+                          agents: np.spec.replicas,
```
Comment (celdrake) [inline src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx:156]: I'm missing a lot of context, but is it correct that `agents` and `count` are the same?

**Review by celdrake — APPROVED** [review]

File: src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx (hunk 1 above)
Comment (rawagner) [reply src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx:156]: good catch, that is definitely wrong!

**Review by celdrake — APPROVED** [review]

File: src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+                            }
+                          : undefined
+                      }
+                    />
+                    <Td>{np.metadata?.name}</Td>
+                    <Td>
+                      <Popover
+                        aria-label="node pool hosts"
+                        hasAutoWidth
+                        bodyContent={t('{{agents}} of {{count}} host requested is available', {
```
Comment (ammont82) [inline src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx:155]: Please add 'ai:' prefix to strings thtat have to be translated

File: src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+                          agents: nodePoolAgents.length,
+                          count: np.spec.replicas,
+                        })}
+                      >
+                        <Label variant="outline">
+                          {t('{{count}} host requested', { count: np.spec.replicas })}
```
Comment (ammont82) [inline src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx:161]: Please add 'ai:' prefix to strings thtat have to be translated

**Review by ammont82 — CHANGES_REQUESTED** [review]

File: src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx (hunk 3 above)
Comment (rawagner) [reply src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx]: added

File: src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx (hunk 2 above)
Comment (rawagner) [reply src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx]: added

**Review by ammont82 — APPROVED** [review]

**Pull request merged**